Apps/RaptureAPIServer/src/main/resources/rapture/config/rapture.cfg

```diff
@@ -2,3 +2,4 @@ ServerType: "webserver"
 Categories: "alpha"
```

Comment:
ok so wipe out everything in RaptureAPI/src/main/resource/rapture/config and if a user wants to override the defaults then they _must_ specify the env variable?